content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Installing External Control Plane [experimental]
+description: Install external control plane and remote cluster.
+weight: 46
```

Comment:
Why is weight 46? Should probably go to the end (weight: 80)

Reply:
Updated!